components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -1955,18 +1960,9 @@ void AdsServiceImpl::OnURLRequest(
 
 void AdsServiceImpl::OnMaybeServeInlineContentAd(
```

Comment:
why do we need `OnMaybeServeInlineContentAd` at all now?

Reply:
I guess I was confused because it looked like `MaybeServeInlineContentAd` was added to BatAdsImpl, but I guess it was just moved inside the file

Reply:
Fixed by removing decl order change from this PR